hackathon_site/event/jinja2/event/landing.html

```diff
@@ -232,11 +232,11 @@ <h3 class="center white-text">FAQ</h3>
     </div>
     <br /><br />
 </div>
-<!--<div class="ombreBackgroundDark2">-->
-<!--    <div class="container section scrollspy" id="sponsors">-->
-<!--        <div class="row">-->
-<!--            <h3 class="center white-text">Sponsors</h3>-->
-<!--            <div class="row sponsorImgs">-->
+<div class="ombreBackgroundDark2">-->
```

Comment:
Comments in html end the line with `-->`, all of those need to be removed (highlighting lines and doing ctrl + / in any good IDE should comment/uncomment them automatically)